docs/apidoc_intro.ejs.md

```diff
@@ -112,3 +113,21 @@ Use the following environment variables to configure the service when run:
 * `LEDGER_AMOUNT_SCALE` (default: `2`) the number of digits allowed in amounts to the right of the decimal place
 * `LEDGER_LOG_LEVEL` (default: `info`) the allowed levels in order of verbosity are `fatal`, `error`, `warn`, `info`, `debug`, and `trace`
 * `LEDGER_RECOMMENDED_CONNECTORS` (default: `'*'`) a comma-delimited list of connector usernames
+* `LEDGER_SECRET` (default: random bytes) a secret used to sign `/auth_token` tokens. Any length.
+
+## RPC Error Codes
+<a id='rpc_error_codes'></a>
+
+For errors which have an equivalent in HTTP, we choose a five digit error code where the first three digits correspond to the HTTP status code.
```

Comment:
Yes, in the websocket channel. The negative codes are specified in http://www.jsonrpc.org/specification.